### ISSUE TITLE
Build script does not work

### ISSUE THREAD

**Issue by ivelten** [fsharp/FAKE#2065]
### Description

Build script returns an error after following steps of the [Quick Start Guide](https://safe-stack.github.io/docs/quickstart/).

I am usnig Windows 10, with .NET Core SDK 2.1, Node 8.11.3, Yarn 1.9.4 and FAKE 5.0.0-beta008 (installed through chocolatey).

### Repro steps

1. Create a directory somewhere in the machine and run command prompt inside it.

2. Install the template via `dotnet new -i SAFE.Template`

3. Create a new project via `dotnet new SAFE`

4. Run `fake run build.fsx --target run`

### Expected behavior

As the guide says, after a short time, I should have a basic SAFE application running in my browser.

### Actual behavior

Command line tool returns me an error:

```
Script failed with
-> You cannot use the netcore version of FAKE as drop-in replacement, please add a dependencies section (and read the migration guide).
```

Is anyone else having this issue?

**Comment by matthid** [comment]
Can you try upgrading to a more recent version of the stable release? Current is 5.4

**Comment by ivelten** [comment]
Yeah, that was the problem. I updated it and now it's working fine.

**Comment by matthid** [comment]
Any idea why you got that version? Can/should we change a default somewhere?

**Comment by matthid** [comment]
@arthis can you run `fake --version`? I feel like you somehow got an outdated version and the output should give an hint from where that is...

/cc @theimowski for the safe template.

**Comment by arthis** [comment]
```
D:\xxxx\Code\safeExample>fake --version
FakePath: C:\ProgramData\chocolatey\lib\fake\bin\Fake.Runtime.dll
FAKE - F# Make "5.0.0"
```
ok got it , it might be the chocolatey version I installed a few weeks ago.. 

I did a  :

```
D:\xxxx\Code\safeExample>choco update fake
Chocolatey v0.10.11

DEPRECATION NOTICE - choco update is deprecated and will be removed or
 replaced in version 1.0.0 with something that performs the functions
 of updating package indexes. Please use `choco upgrade` instead.
Upgrading the following packages:
fake
By upgrading you accept licenses for the packages.

You have fake v5.0.0-alpha015 installed. Version 5.10.1 is available based on your source(s).
Progress: Downloading fake 5.10.1... 100%

fake v5.10.1 [Approved]
```
the following  works susccefully afterwards :

`D:\xxxx\Code\safeExample>fake build --target run`

thanks a lot for your question and your help.

**Comment by theimowski** [comment]
This issue is caused by the fact that we decided to recommend Fake as global tool. 
I'll add a note to the docs for required version of fake >= 5.10
